cluster/saltbase/salt/README.md

```diff
@@ -0,0 +1,31 @@
+This directory forms the base of the main SaltStack configuration. The
+place to start with any SaltStack configuration is
+[top.sls](top.sls). However, unless you are particularly keen on
+reading Jinja templates, the following tables break down what
+configurations run on what providers:
+
+Key: M = Config applies to master, n = config applies to nodes
+
+Config                                              | GCE   | Vagrant | AWS | Azure
+----------------------------------------------------|-------|---------|-----|------
```

Comment:
Cold you please add a section for `_states/`, or a brief desc. as to how it's included?